movienight.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "movienight",
+    "plugin_schema": "2",
+    "release": "12.1-RELEASE",
+    "artifact": "https://github.com/zorglube/iocage-plugin-movienight.git",
+    "official": false,
+    "properties": {
+        "dhcp": 1
+    },
+    "pkgs": [
+        "nano", "bash", "gzip", "ca_root_nss", "git", "lang/go"
```

I can not say there is anything wrong here, but I have a few more suggestions to consider (sorry I was viewing from my phone last time, so I skipped over this)

- Using `git-lite` can save a few MB on the download
- Seems like other plugin manifest only list one package per line (bonus points if you list them alphabetically)

```suggestion
        "nano",
        "bash",
        "gzip",
        "ca_root_nss",
        "git-lite",
        "go"
```

---

Done, thank for the idea.